src/main/java/lk/gov/govtech/covid19/util/Constants.java

```diff
@@ -8,13 +8,11 @@
     public static final String DHIS_API_CONTEXT = "/dhis";
 
     /*
-    * context for all calls related to web portal used for notification publication
+    * context for all calls related to web portal UI used for notification publication
     * */
     public static final String PORTAL_API_CONTEXT = "/portal";
-    //sub context for news notification publication
-    public static final String NEWS_API_CONTEXT = "/news";
-    //sub context for case notification publication
-    public static final String CASE_API_CONTEXT = "/cases";
+    public static final String NEWS_PATH = "/news";
```

Comment:
any reason to change the naming convention?

Reply:
Because they are within the portal context. e.g. /portal/news and /portal/cases To not get confused with the main ones. Should I stick to the previous?

Reply:
Any suggestions one what to name them?

Reply:
no you can have the context as whatever. I meant the constant name :)